tests/integration/test_dynamodb.py

```diff
@@ -271,6 +271,33 @@ def test_empty_and_binary_values(self):
         response = table.put_item(Item=item2)
         self.assertEqual(response['ResponseMetadata']['HTTPStatusCode'], 200)
 
+    def test_dynamodb_stream_shard_iterator(self):
+        dynamodb = aws_stack.connect_to_service('dynamodb')
+        ddbstreams = aws_stack.connect_to_service('dynamodbstreams')
+
+        table_name = 'table_with_stream'
```

Comment:
nit: Would be better to use a random name (using `short_uid()`) here - and delete the table at the end of the test method. (Shouldn't hold up the merge, though. Can be tackled in a future iteration.)